deploy/yaml/namespace.yaml

```diff
@@ -0,0 +1,4 @@
+apiVersion: v1
```

Comment:
I think you can write the yaml that creates the kmesh-system in `kmesh.yaml`.

refer to https://github.com/istio/istio/blob/47d9c98ee413768a3e311dc238703d10e610bfb4/pkg/config/analysis/analyzers/testdata/sidecar-default-selector.yaml#L100